src/isIsogram.test.js

```diff
@@ -6,4 +6,27 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for a word with no repeating letters', () => {
+    expect(isIsogram('playgrounds')).toBe(true);
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    expect(isIsogram('look')).toBe(false);
+  });
+
+  it('should return false for a case-sensitive repeating letters', () => {
+    expect(isIsogram('Adam')).toBe(false);
```

Comment:
The test case for 'Adam' should expect `true` instead of `false` because the function is supposed to be case-insensitive. According to the task description, 'Adam' should be considered an isogram.